generate.js

```diff
@@ -171,32 +155,13 @@ const createRoom = () => {
       } else {
         bedDetails = `${noOfBed} ${typeOfBEd} ` + 'beds';
       }
-      dataItem.sleeping_arrangements.push(bedDetails);
+      word += `${bedDetails}* `;
     }
-
-    const newRoom = new Room(dataItem);
-    const temp = newRoom.save();
-    data.push(temp);
+    out.write(`${word}\n`);
   }
 
-  Promise.all(data)
-    .then(results => {
-      console.log(`${results.length} data saved in dB!`);
-    })
-    .catch(err => {
-      console.error(err);
-    })
-    .then(() => {
-      mongoose.connection.close(() => {
-        process.exit(0);
-      });
-    });
 };
 
-db.Room.remove({}).exec(function(err, results) {
-  if (err) {
-    console.error(err);
-    return process.exit(0);
-  }
-  createRoom();
-});
+const out = fs.createWriteStream('./test.csv');
```

Comment:
You might want to move this to the top of your file (before you call out.write) so it's easier to see what out is